flink-python/pyflink/fn_execution/beam/beam_coders.py

```diff
@@ -354,3 +355,29 @@ def _create_impl(self):
         HIGHEST_PROTOCOL = pickle.HIGHEST_PROTOCOL
         return coder_impl.CallbackCoderImpl(
             lambda x: dumps(filter_data_views(x), HIGHEST_PROTOCOL), pickle.loads)
+
+
+class BeamDataStreamStatefulMapCoder(FastCoder):
```

Comment:
Is it possible to reuse BeamDataStreamStatelessFlatMapCoder? Maybe need to refactor the name a bit.

Reply:
Yes, actually, the BeamDataStreamStatefulMapCoder is the extension of BeamDataStreamStatelessFlatMapCoder, in order to distinguish stateful and stateless operation.